src/main/resources/case_handed_offline.bpmn

```diff
@@ -14,59 +12,104 @@
     <bpmn:serviceTask id="CaseHandedOfflineDefendantTaskId" name="Send email respondent solicitor 1" camunda:type="external" camunda:topic="processCaseEvent">
       <bpmn:extensionElements>
         <camunda:inputOutput>
-          <camunda:inputParameter name="CCD_ID">1600350694715989</camunda:inputParameter>
-          <camunda:inputParameter name="CASE_EVENT">NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE</camunda:inputParameter>
+          <camunda:inputParameter name="caseId">1601986692564009</camunda:inputParameter>
+          <camunda:inputParameter name="caseEvent">NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE</camunda:inputParameter>
```

Comment:
still not convinced by this hardcoding, need to remember to remove when we can trigger events from submitted callbacks

Reply:
Only CaseId will be passed from Submitted callabck, however CaseEvent will always be there